libs/input-mapping/src/Helper/SourceRewriteHelper.php

```diff
@@ -61,9 +61,11 @@ private static function rewriteSource(string $source, ClientWrapper $clientWrapp
     {
         $newSource = self::getNewSource(
             $source,
-            $clientWrapper->getBasicClient()->webalizeDisplayName((string) $clientWrapper->getBranchId())['displayName']
+            $clientWrapper->getBranchClientIfAvailable()->webalizeDisplayName(
+                (string) $clientWrapper->getBranchId()
+            )['displayName']
```

Comment:
Tady bych rekl, ze se muze pouzit i BasicClient na tu webalizaci? Ale asi je to jedno.

Kazdopadne chapu to tak, ze ten fallback se vlastne vubec nezmenil? Nebo se jeste bude menit? 

Nejak mi nedochazi, jak to, ze to funguje stejne pro "fake" dev Storage i pro skutecne branchovanou Storage :)

Reply:
může, u webalize se může použít cokoliv - záměr byl zlikvidovat co nejvíc volání getBasicClient(), protože by se měl používat jen tam kde, je to opravdu nutný.

Fallback se ještě měnit bude.

Reply:
Fallback se jeste menit bude - OK, tak pak mne to takhle prijde OK. 

Basic client se teda pouzije jen v tom DevBranches klientovi. Uz chapu proc ho tady nechces mit. Jasne.

Reply:
Jo, potomhle by to mělo
- v normálním projektu fungovat beze změny
- v SOX projektu (po nastavení useBranchStorage flagu https://github.com/keboola/job-runner/pull/222/files#diff-14dce51b474644e4b5ec8d720956607f343651075696af47d9398824d8693345L32) fungovat tak, že to čte/zapíše tabulky do/z DEV storage (správně), ale pořád ještě se prefixem (špatně)

Reply:
Taky bych řekl že tu má být basic client ne? Jinak ti to v soxu bude volat branchovanej endpoint  a tenhle imho není?

Reply:
Extra jsem se na to ptal https://keboolaglobal.slack.com/archives/C05BK5V8N1Z/p1688724311762769?thread_ts=1688723005.478619&cid=C05BK5V8N1Z

Reply:
Aha super. To jsem nevěděl 👍